src/hooks/useForm.js

```diff
@@ -35,6 +35,12 @@ const useForm = (callback, layout, customComponents) => {
         'displaytext': DisplayText,
     }
 
+    useEffect(() => {
+        if (_.size(_.omit(initValues, _.identity)) > 0) {
+            setValues(initValues)
+        }
+    }, [initValues])
```

Comment:
Ik had half verwacht dat als initValues verandert, de `useState(initValues)` al vanzelf opnieuw triggert

Reply:
Nee dat gebeurt niet, want de waarde die je in de `useState` hook stopt wordt alleen gebruikt bij de initiele render.